packages/contract-wrappers/test/utils/token_utils.ts

```diff
@@ -1,33 +1,45 @@
-import * as _ from 'lodash';
+import { generatePseudoRandomSalt } from '@0xproject/order-utils';
+import { BigNumber } from '@0xproject/utils';
 
-import { InternalContractWrappersError, Token } from '../../src/types';
+import { artifacts } from '../../src/artifacts';
+import { DummyERC721TokenContract } from '../../src/contract_wrappers/generated/dummy_erc721_token';
 
-const PROTOCOL_TOKEN_SYMBOL = 'ZRX';
-const WETH_TOKEN_SYMBOL = 'WETH';
+import { constants } from './constants';
+import { provider, txDefaults, web3Wrapper } from './web3_wrapper';
 
-export class TokenUtils {
-    private _tokens: Token[];
-    constructor(tokens: Token[]) {
-        this._tokens = tokens;
-    }
-    public getProtocolTokenOrThrow(): Token {
-        const zrxToken = _.find(this._tokens, { symbol: PROTOCOL_TOKEN_SYMBOL });
-        if (_.isUndefined(zrxToken)) {
-            throw new Error(InternalContractWrappersError.ZrxNotInTokenRegistry);
-        }
-        return zrxToken;
-    }
-    public getWethTokenOrThrow(): Token {
-        const wethToken = _.find(this._tokens, { symbol: WETH_TOKEN_SYMBOL });
-        if (_.isUndefined(wethToken)) {
-            throw new Error(InternalContractWrappersError.WethNotInTokenRegistry);
-        }
-        return wethToken;
-    }
-    public getDummyTokens(): Token[] {
-        const dummyTokens = _.filter(this._tokens, token => {
-            return !_.includes([PROTOCOL_TOKEN_SYMBOL, WETH_TOKEN_SYMBOL], token.symbol);
-        });
-        return dummyTokens;
-    }
-}
+const DUMMY_ERC_20_ADRESSES = [
+    '0x07f96aa816c1f244cbc6ef114bb2b023ba54a2eb',
+    '0x6a4a62e5a7ed13c361b176a5f62c2ee620ac0df8',
+    '0x6dfff22588be9b3ef8cf0ad6dc9b84796f9fb45f',
+    '0xcfc18cec799fbd1793b5c43e773c98d4d61cc2db',
+    '0xf22469f31527adc53284441bae1665a7b9214dba',
```

Comment:
Where are these hard-coded addresses coming from?

Reply:
Migrations. They're deterministic. Before we were fetching that from a token registry. Now it doesn't exist

Reply:
Let's add a comment above to that effect.